### ISSUE TITLE
A VAMC editor re-assigns a facility health service, it creates an orphan in lighthouse.

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#4915]
**Describe the defect**

[We may not want this to happen at all anyway]

Our approach to pushing COVID Vaccine services (and later all services) works fine in normal CRUD operations and workflows.  However if an editor assigns a service to one facility and saves it (causing a push to Lighthouse) and then re-assigns the service to another facility (causing a push to the new service) does not also push an `active: false` to the original facility, so it creates an untracked orphan in Lighthouse.

The risk for this happening with the covid data is low, since it is currently being managed by one editor, but when it is opened up to multiple editors or we are pushing service data for more services, this will be much more likely to happen.

**To Reproduce**
1. create a new COID vaccine service and assign it to a facility A.
2. Click save.  (workflow state does not matter) (an item will be queued for facility A)
3. Edit the service to set a different facility B
4. Click save.  (workflow state does not matter) (an item will be queued for facility B)
5.  After push, Lighthouse will now have COvid data for both A and B.

**Expected behavior**
1. create a new COID vaccine service and assign it to a facility A.
2. Click save.  (workflow state does not matter) (an item will be queued for facility A)
3. Edit the service to set a different facility B
4. Click save.  (workflow state does not matter) 
  - an item will be queued for facility B
  - an item will be queued for facility A setting `active:false`
5.  After push, Lighthouse will now have COVID data for B.

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS objective (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

**Comment by jilladams** [comment]
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11367 might have resolved this. To verify.

**Comment by kmariepat-cityfriends** [comment]
Confirmed #14136 resolves the issue stated on this card once that work is complete

**Comment by jilladams** [comment]
We probably need to re-verify whether or not it is (with Christian).

**Comment by Agile6MSkinner** [comment]
@omahane Is this still relevant?
